src/Novell.Directory.Ldap.NETStandard/Connection.cs

```diff
@@ -825,7 +825,7 @@ internal MessageAgent GetMessageAgent(int msgId)
         /// </param>
         internal void RemoveMessage(Message info)
         {
-            SupportClass.VectorRemoveElement(_messages, info);
```

Comment:
you can also remove this leftover - SupportClass.VectorRemoveElement

Reply:
Raised https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/pull/146